assets/js/aplus.js

```diff
@@ -666,27 +670,18 @@ $(function() {
  * Note! Translation requires that at least 'hl' query parameter is copied
  */
 (function(window) {
-    window.addEventListener('click', function(event) {
-        if (
-            event.target.nodeName === 'A'
-            && event.target.href
-            && event.target.protocol === window.location.protocol
-            && event.target.host === window.location.host // hostname:port
-            && (event.target.search === '' || event.target.search === '?') // no search
-        ) {
-            event.preventDefault(); // stop default redirect
-            event.target.search = window.location.search; // copy query params
-            if (event.target.getAttribute('target') === '_blank') {
-                // Open a new window or tab.
-                window.open(
-                    event.target.href,
-                    '_blank',
-                    'location,menubar,resizable,scrollbars,status,noopener');
-            } else {
-                window.location.href = event.target.href; // redirect the current window
-            }
-        }
-    }, false);
+  const addQueryParameter = (event) => {
+  if (
+      event.target.protocol === window.location.protocol
+      && event.target.host === window.location.host // hostname:port
+      && window.location.search.includes('?hl=en') // hl search
+  ) {
+      event.target.search = window.location.search; // copy query params
+  }
+}
+
+$(document).on('click', 'a[href]', addQueryParameter);
+
```

Comment:
Does the new `<a>` event handler let the normal click handler work too? The `_blank` target if branches were removed, but I suppose the standard event handler would take care of it (open the link in a new window or the same window).

Reply:
This event handler does not overwrite any other event handler. This is basically one event handler on top of the existing ones.